app/models/setup/affect_relation_methods.rb

```diff
@@ -38,7 +38,7 @@ def affects_to(model)
     def collect_models_from(constant_names)
       models = []
       constant_names.each do |model_name|
-        if model = model_name.constantize rescue nil
+        if ((model = model_name.constantize).present? rescue nil)
```

Comment:
I think it should be
    if ((model = model_name.constantize).present? rescue false)
or
   if ((model = model_name.constantize) rescue nil).present?